### ISSUE TITLE
Page props not updated on query paremeters change

### ISSUE THREAD

**Issue by lambrojos** [postor/next-i18n-helper#1]
Hi, first of all thanks for your work - this  has taken me out of a really problematic situation :)
Anyway, I have observed that a wrapped page, if reached consecutive times with different parameters, won't update its initial props, for example:

Suppose we have a page accepting a query parameter, ( /video?id=123)
If inside the page there is a router link to another video (video?id=567), if I click it the page will render again, but the props wont update - the id in the address bar will be 567, but I will still be seeing the video with id 123.

I have tracked this down to here:
https://github.com/nextjs-boilerplate/next-i18n-helper/blob/master/src/wrapper.js#L44

If the same page gets rendered two times with different initial props, `this.pageInitialProps` the second time wont contain the updated properties. 

I found out that update properties on subsequent loads end up in `this.props.pageInitialProps`, so modifying the Wrapper class render method in 

```
render () {
    this.pageInitialProps = Object.assign(this.pageInitialProps, this.props.pageInitialProps)
    return <I18nextProvider i18n={this.i18n}>
      <Page {...this.pageInitialProps} />
    </I18nextProvider>
  }
```
actually solved my problem.

Is there a better way to do it? Or, If you are intereseted I can provide an example and/or a PR

thanks

**Comment by postor** [comment]
Thank you, can you give me a push?

**Comment by postor** [comment]
Fixed in version 1.0.18, wait for your feedback @lambrojos

**Comment by lambrojos** [comment]
Sorry I didn had much time - anyway today I'll give it a try
